### PR TITLE
Update copyright to 2022.

### PR DESCRIPTION
**Pull request by cqc-alec**
Did:
`git grep -l "2021 Cambridge Quantum Computing" | xargs sed -i 's/2021 Cambridge Quantum Computing/2022 Cambridge Quantum Computing/g'`

### CODE REVIEW

File: modules/pytket-qsharp/pytket/extensions/qsharp/backends/config.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Cambridge Quantum Computing
+# Copyright 2022 Cambridge Quantum Computing
```
Comment (cqc-melf) [inline modules/pytket-qsharp/pytket/extensions/qsharp/backends/config.py:1]: This should be 2021-2022?

File: modules/pytket-qsharp/pytket/extensions/qsharp/backends/azure_quantum.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Cambridge Quantum Computing
+# Copyright 2022 Cambridge Quantum Computing
```
Comment (cqc-melf) [inline modules/pytket-qsharp/pytket/extensions/qsharp/backends/azure_quantum.py:1]: This should be 2021-2022?

File: modules/pytket-qsharp/tests/azure_test.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Cambridge Quantum Computing
+# Copyright 2022 Cambridge Quantum Computing
```
Comment (cqc-melf) [inline modules/pytket-qsharp/tests/azure_test.py:1]: This should be 2021-2022?

File: modules/pytket-qiskit/pytket/extensions/qiskit/backends/config.py [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Cambridge Quantum Computing
+# Copyright 2022 Cambridge Quantum Computing
```
Comment (cqc-melf) [inline modules/pytket-qiskit/pytket/extensions/qiskit/backends/config.py:1]: This should be 2021-2022?

File: modules/pytket-braket/pytket/extensions/braket/backends/config.py [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Cambridge Quantum Computing
+# Copyright 2022 Cambridge Quantum Computing
```
Comment (cqc-melf) [inline modules/pytket-braket/pytket/extensions/braket/backends/config.py:1]: This should be 2021-2022?

File: modules/pytket-qsharp/pytket/extensions/qsharp/backends/config.py (hunk 1 above)
Comment (cqc-alec) [reply modules/pytket-qsharp/pytket/extensions/qsharp/backends/config.py:1]: Oops, yes thanks.

**Review by cqc-melf — APPROVED** [review]

**Pull request merged**